system_tests/staker_test.go

```diff
@@ -67,8 +67,7 @@ func stakerTestImpl(t *testing.T, faultyStaker bool, honestStakerInactive bool)
 	if faultyStaker {
 		l2info.GenerateGenesysAccount("FaultyAddr", common.Big1)
 	}
-	sequencerTxOpts := l1info.GetDefaultTransactOpts("Sequencer", ctx)
-	l2clientB, l2nodeB := Create2ndNodeWithConfig(t, ctx, l2nodeA, l1stack, &l2info.ArbInitData, arbnode.ConfigDefaultL1Test(), &sequencerTxOpts)
+	l2clientB, l2nodeB := Create2ndNodeWithConfig(t, ctx, l2nodeA, l1stack, l1info, &l2info.ArbInitData, arbnode.ConfigDefaultL1Test())
```

Comment:
I just tried it. The AddSequencerL2BatchFromOrigin0 fails with:
"full_challenge_impl_test.go:153:  [] execution reverted "
I'll have a look.
This change shouldn't make it fail, because the Create1ndNodeWithConfig still uses sequencer tx opts, only it now happends inside the function